src/main/java/de/uni_tuebingen/ub/nppm/servlet/gast/StatistikServlet.java

```diff
@@ -0,0 +1,43 @@
+package de.uni_tuebingen.ub.nppm.servlet.gast;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class StatistikServlet extends AbstractGastServlet {
+
+    @Override
+    protected String getNavigationTitle() {
+        return "stat";
+    }
+
+    @Override
+    protected String getTitle() {
+        return "Statistik";
+    }
+
+    @Override
+    protected void generatePage(HttpServletRequest request, HttpServletResponse response) throws Exception {
```

Comment:
I don't think we will need this in any other case except Quelle. Can we integrate this into the existing quelle servlet somehow? Otherwise we might rename it to QuelleStatistikServlet

Reply:
Yes i have included it in the Quellen Servlet